e2e_tests/helpers.py

```diff
@@ -83,6 +75,24 @@ async def post_resource(payload, endpoint, resource_type, token, admin_token, ve
         return resource_path, resource_id
 
 
+async def get_shared_service_id_by_name(template_name: str, verify, token):
+    async with AsyncClient(verify=verify, timeout=30.0) as client:
```

Comment:
use the same `timeout` value here as the others?

Reply:
Good catch, missed that!